pickupDate.java

```diff
@@ -0,0 +1,66 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class pickupDate {
```

Comment:
Classnames should start with a capital letter! And as this class contains more than just the pickupDate maybe better rename it to something else, like (TaxiRecord)
```suggestion
public class TaxiRecord {
```

Reply:
OK！